openzwave-sys/build.rs

```diff
@@ -6,10 +66,28 @@ fn main() {
      .file("openzwave-c/manager.cc")
      .file("openzwave-c/notification.cc")
      .file("openzwave-c/value_classes/value_id.cc")
-     .include("/usr/include/openzwave") // workaround for https://github.com/OpenZWave/open-zwave/issues/771
-     .include("/usr/local/include/openzwave")
      .cpp(true)
-     .flag("-std=c++11") // to iterate with ranges
-     .compile("libopenzwave-c.a");
-    println!("cargo:rustc-flags=-l openzwave");
+     .flag("-std=c++11"); // to iterate with ranges
+
+    if find_include_path(&mut c).is_ok() {
```

Comment:
We shouldn't use pkg-config if we're building open-zwave as a sub-module. We know exactly where everything is.